src/vs/workbench/browser/parts/editor/breadcrumbs.ts

```diff
@@ -90,10 +90,18 @@ export abstract class BreadcrumbsConfig<T> {
 					readonly name = name;
 					readonly onDidChange = onDidChange.event;
 					getValue(overrides?: IConfigurationOverrides): T {
-						return service.getValue(name, overrides);
+						if (overrides) {
```

Comment:
@sandy081 is that really needed?

Reply:
`service.getValue(name, overrides)` this should be good enough. Does the TS complains when using this?

Reply:
I see.. looks like the API is strict here to not to accept `undefined` value for overrides. I can relax it if needed or you can go with above if/else branch.

Reply:
please relax @sandy081